docs/api/index.md

```diff
@@ -760,6 +766,7 @@ scrollBehavior?: ScrollBehavior
 function scrollBehavior(to, from, savedPosition) {
   // `to` and `from` are both route locations
   // `savedPosition` can be null if there isn't one
+  return savedPosition || { top: 0 };
```

Comment:
this is just the API, let's remove this

```suggestion
```

Reply:
Fair enough, since the heading was "example" I thought it might be helpful to show a return value. Removing.